ENIGMAsystem/SHELL/Platforms/SDL/Info/platform_info.h

```diff
@@ -0,0 +1,5 @@
+// Informative header designed to grant superior control over platform-
+// or API-dependent behavior. This file can define any number of macros
+// describing various compatibility and feature points.
+
+#define ENIGMA_WS_SDL 1
```

Comment:
We need to retire this shit... I think the things that were using this info stopped, and they're certainly not using new info.